packages/hooks/src/use_context.rs

```diff
@@ -22,6 +34,24 @@ pub fn use_context<T: 'static + Clone>() -> T {
 /// Provide some context via the tree and return a reference to it
 ///
 /// Once the context has been provided, it is immutable. Mutations should be done via interior mutability.
+/// Context can be read by any child components of the context provider, and is a solution to prop
+/// drilling, using a context provider with a Signal inside is a good way to provide global/shared
+/// state in your app:
+/// ```rust
+///fn app() -> Element {
+///    use_context_provider(|| Signal::new(0));
+///    rsx! { Child {} }
+///}
+// This component does read from the signal, so when the signal changes it will rerun
```

Comment:
Comments are a bit odd within doc comments. I think this needs to be `/// // This...` if you want the comment to show up in the docs